wire-schema/src/main/java/com/squareup/wire/schema/IdentifierSet.java

```diff
@@ -78,13 +86,26 @@ public boolean includes(ProtoMember protoMember) {
   private boolean includes(String identifier) {
     if (includes.isEmpty()) return !exclude(identifier);
 
-    boolean included = false;
+    String includeMatch = null;
+    String excludeMatch = null;
     while (identifier != null) {
-      if (excludes.contains(identifier)) return false;
-      included = included || includes.contains(identifier);
+      if (excludes.contains(identifier)) {
+        excludeMatch = identifier;
+      }
```

Comment:
Hold up I'm thinking about this backwards. We're looking at an identifier and checking in a set of rules.